<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1517</width>
    <height>784</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Experiment Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>18</y>
      <width>92</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QFrame" name="horizontalFrame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>841</width>
      <height>698</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QTableWidget" name="current_table_listWidget">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>308</y>
      <width>541</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>80</number>
    </attribute>
    <column>
     <property name="text">
      <string>time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>units</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>x1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>y1</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="timeGroupBox">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>180</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Time point</string>
    </property>
    <widget class="QGroupBox" name="timeMeasureGroupBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>121</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="secondsRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>seconds</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="minutesRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>minutes</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="hoursRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>hours</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="daysRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>days</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="add_current_data_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_line_from_current_list_pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>51</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Del</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="timeNumberDoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="currentFile_const_label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>698</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current file: </string>
    </property>
   </widget>
   <widget class="QLabel" name="currentFile_label">
    <property name="geometry">
     <rect>
      <x>865</x>
      <y>698</y>
      <width>581</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="showResults_pushButton">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>20</y>
      <width>21</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Results</string>
    </property>
   </widget>
   <widget class="QListWidget" name="opend_files_listWidget">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>20</y>
      <width>225</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QLabel" name="opened_files_const_label">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>0</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Opened files:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="add_options_groupBox">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>60</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Add options</string>
    </property>
    <widget class="QRadioButton" name="add_exp_point_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add exp. point</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="add_norm_value_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>58</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Add norm. value</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="add_time_point_radioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Add time point</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="coord_label">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>111</width>
      <height>54</height>
     </rect>
    </property>
    <property name="text">
     <string>x: - 
y: -</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Experiment</string>
    </property>
    <widget class="QPushButton" name="delete_experiment_pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/delete icon.png</normaloff>:/delete icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="show_x_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>93</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Show x</string>
     </property>
    </widget>
    <widget class="QPushButton" name="change_experiment_name_pushButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/change icon.png</normaloff>:/change icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
    <widget class="QCheckBox" name="use_normalization_checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Normalize</string>
     </property>
    </widget>
    <widget class="QPushButton" name="add_new_experiment_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/add icon.png</normaloff>:/add icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>34</width>
       <height>34</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="normalization_tableWidget">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>308</y>
      <width>95</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="autoScrollMargin">
     <number>14</number>
    </property>
    <column>
     <property name="text">
      <string>norm.</string>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="lines_on_chart_listWidget">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>20</y>
      <width>60</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lines_const_label">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>0</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lines:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_line_pushButton">
    <property name="geometry">
     <rect>
      <x>923</x>
      <y>230</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/arrow_down icon.png</normaloff>:/arrow_down icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_line_pushButton">
    <property name="geometry">
     <rect>
      <x>923</x>
      <y>200</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/arrow_up icon.png</normaloff>:/arrow_up icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="turn_on_all_lines_pushButton">
    <property name="geometry">
     <rect>
      <x>923</x>
      <y>270</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/refresh icon.png</normaloff>:/refresh icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1517</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuChart">
    <property name="title">
     <string>Chart</string>
    </property>
    <widget class="QMenu" name="menuChange_Y_axis_name_3">
     <property name="title">
      <string>Change Y axis name</string>
     </property>
     <addaction name="actionAmplitude_2"/>
     <addaction name="actionReflection"/>
     <addaction name="actionAbsorbtion"/>
    </widget>
    <addaction name="actionChange_axes_range"/>
    <addaction name="menuChange_Y_axis_name_3"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionSave_to_file"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionAuto_cell_switching"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConvert_file_Oscilloscope"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuChart"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionChange_axes_range">
   <property name="text">
    <string>Change axes range</string>
   </property>
  </action>
  <action name="actionSave_to_file">
   <property name="text">
    <string>Save to file</string>
   </property>
  </action>
  <action name="actionAmplitude_2">
   <property name="text">
    <string>Amplitude</string>
   </property>
  </action>
  <action name="actionReflection">
   <property name="text">
    <string>Reflection</string>
   </property>
  </action>
  <action name="actionAbsorbtion">
   <property name="text">
    <string>Absorbtion</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file (Ocean Optics)</string>
   </property>
  </action>
  <action name="actionAuto_cell_switching">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="applications-other">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Auto cell switching</string>
   </property>
  </action>
  <action name="actionConvert_file_Oscilloscope">
   <property name="text">
    <string>Convert file (Oscilloscope)</string>
   </property>
  </action>
  <action name="actionOpen_file_excel">
   <property name="text">
    <string>Open file (Excel)</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
